packages/core/src/components/axes/axis.ts

```diff
@@ -224,6 +224,7 @@ export class Axis extends Component {
 		// create the right ticks formatter
 		let formatter: any
 		const userProvidedFormatter = getProperty(axisOptions, 'ticks', 'formatter')
+		const { code, number: numberFormatter } = getProperty(options, 'locale')
```

Comment:
```suggestion
		const { code: localeCode, number: numberFormatter } = getProperty(options, 'locale')
```

Reply:
pls use more readable names. it's clear on this line that you're pulling a locale code, but in another line in this file when you will pass `code` into a function, it's not clear (from the name) what `code` is

Reply:
ok I'll keep this in mind, I have updated to localeCode.